src/test/java/teammates/test/cases/ui/browsertests/InstructorFeedbackEditPageUiTest.java

```diff
@@ -96,6 +96,7 @@ private void testGeneralQuestionOperations() throws Exception {
 
         testEditQuestionLink();
         testEditQuestionAction();
+        testEditVisibilityAction();
```

Comment:
Preferably the new test should be in the same place as the older similar tests. But if you really want to go with a new method, you should change the name as you suggested.